hardware_accelerators/das-about-dynamic-accelerator-slicer-operator.adoc

```diff
@@ -0,0 +1,92 @@
+
+For more information, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope]. 
+====
+
+Running data intensive workloads on the platform requires efficient GPU utilization to offset the high cost of the resources involved. One approach for maximizing efficiency is through the use of oversubscription of the GPU resources by time-slicing multiple workloads. This method provides for sharing GPU resources by alternating the execution times of the workloads.
+
+This approach allows you to define static default time-slicing configurations, but it can lead to resource wastage if the slicing does not align with the actual workload demands. Additionally, changing static partitions created at node boot-time requires the eviction of all the workloads at the node level in order to create a new set of partitions. 
+
+The Dynamic Accelerator Slicer (DAS) Operator allows you to optimize GPU utilization for workloads on {product-title} through the use of dynamically slicing GPU accelerators in {product-title}, instead of relying on statically sliced GPUs defined when the node is initially started. This allows you to dynamically slice GPUs based on specific workload demands, ensuring efficient resource utilization. 
+
+Dynamic slicing is useful in the following situations:
+
+* You do not know all the accelerator partitions needed on every node on the cluster.
+
+* The partition requirements change at the workload level rather than at the node level.
+
+* You do not want to learn or use new APIs to request accelerators slices.
+
+* You prefer to use stable device plugins APIs for creating partitions.
+
+:FeatureName: Dynamic Accelerator Slicer Operator
+
+The DAS Operator currently includes a reference implementation for NVIDIA Multi-Instance GPU (MIG) and is designed to support additional technologies such as NVIDIA MPS or GPUs from other vendors.
+
+The DAS Operator components interact to provide GPU slice allocation. Pods requesting GPU slices are mutated by a webhook to use the `mig.das.com` extended resource. The scheduler plugin tracks slice availability and creates `AllocationClaim` objects processed by the device plugin on each node.
+
+.Limitations
+
+Keep in mind the following limitations when using the Dynamic Accelerator Slicer Operator.
+
+ * You need to identify potential incompatibilities and ensure the system works seamlessly with various GPU drivers and operating systems.
+
+ * The Operator only works with specific NVIDIA GPUs and drivers (H100, A100).
+
+ * When using the Operator, only the first container of the pod can access the GPU. 
```

Comment:
This is no longer true for the TP release, please remove this line.

Reply:
Removed.